lib/models/ObjectLockConfiguration.ts

```diff
@@ -36,6 +36,7 @@ export type ParsedRetention =
 export default class ObjectLockConfiguration {
     _parsedXml: any;
     _config: Config;
+    _days: number | null;
```

Comment:
how/where `_days` is being used?

Reply:
This was initially added for cases where the object lock configuration retention length would need to be checked, such as the intended epic objective:

> Update putBucketObjectLockConfig to check for the bucket policy and prevent setting retention period outside of the allowed limit of bucket policy

This proved to not be AWS standard, and as such this addition is indeed not used anywhere. I have removed the changes to this file, and updated the[ CS PR](https://github.com/scality/cloudserver/pull/5501). It is passing, both in CI and hand testing.